master/docs/manual/cfg-www.rst

```diff
@@ -119,10 +119,10 @@ The available classes are described here:
 
     ::
 
-        from buildbot.plugins import util
+        from buildbot.www import auth
         c['www'] = {
             # ...
-            'auth': util.UserPasswordAuth({"homer": "doh!"}),
```

Comment:
Yes, the class is in www.auth, but it is also available in plugin.util as per setup.py. This allows us to move the class and be happy knowing the users are supposed to use the plugin namespace to important it.

I agree this can be confusing.